Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.yml

```diff
@@ -35,13 +35,27 @@ configuration:
   name: incident_collections
   options:
   - compromised/account_group
-  - compromised/card
+  - compromised/bank_card_group
   - compromised/breached
-  - bp/phishing
-  - bp/phishing_kit
+  - compromised/mule
   - osi/git_repository
   - osi/public_leak
-  - malware/targeted_malware
+  - osi/vulnerability
```

Comment:
If someone already has `malware/targeted_malware` configured, will it break for them when they upgrade?